components/wasteplan_trv.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "wasteplan_trv",
+    "owner": [
+        "@jonkristian"
+    ],
+    "manifest": "https://raw.githubusercontent.com/jonkristian/wasteplan_trv/master/custom_components/wasteplan_trv/manifest.json",
```

Comment:
This integration does not have any requirements to build wheels for, so it should not be added here.